### ISSUE TITLE
Device orientation is not working on a new empty project

### ISSUE THREAD

**Issue by ptomasroos** [facebook/react-360#740]
### Description

This is a bug. 
Creating a new empty project with

```
react-360 init TestProject 
cd TestProject
yarn run start
```

#### Expected behavior

Navigating to the test project with a phone
And "looking" around by moving the phone around should pan the 360 world.
 
#### Actual behavior

Nothing happens,
Only dragging with finger will pan the world.

### Reproduction

Above

### Solution

I can see there is support in the DeviceOrientationCameraController, i do not know what else is needed.

### Additional Information

* 'react-360' package version: 1.1.0 
* 'react-360-web' package version: 1.1.0
* Operating System: MacOS but Android 9 for tests
* Browser: Chrome 75
* VR Device: NO, just phone

**Comment by jackabe** [comment]
Any update?

**Comment by ptomasroos** [comment]
Are you having the same issue?

**Comment by jackabe** [comment]
Yep! Works on Firefox annoyingly. I looked into the DeviceOrientationCameraController and when I log out the 'enabled' flag it is false. For some reason it comes to the conclusion that my Chrome browser is not supported.

**Comment by andonivianez** [comment]
I´m having the same trouble, do you know how can i fix it?
Thanks!

**Comment by andonivianez** [comment]
Hi!
I´ve "fixed" this, using **https connection** in (for example) Firebase Hosting (you can try it also with Glitch or an dedicated server). It seems like Chrome it´s blocking VR API due to low security connection.
In Safari it´s already not working...I´ve to do more tests.
